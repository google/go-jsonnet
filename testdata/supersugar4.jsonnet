{ assert self.x, y: 42 } { x: true, y +: 1 }
