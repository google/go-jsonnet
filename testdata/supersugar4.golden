{
   "x": true,
   "y": 43
}
